ci-operator/config/openshift/managed-node-metadata-operator/openshift-managed-node-metadata-operator-main.yaml

```diff
@@ -17,14 +11,37 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: generate
-  commands: make generate --always-make && git diff --exit-code
```

Comment:
are we losing anything without this target? i think a basic build is already baked into prow which i hope would include generating necessary sources...

Reply:
Nope, this will be covered by `make validate`

validate target: https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator/standard.mk#L292-L293

generate-check target that gets called through validate: https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator/standard.mk#L266-L271